packages/blade/src/components/Box/__tests__/Box.web.test.tsx

```diff
@@ -119,6 +119,32 @@ describe('<Box />', () => {
     `);
   });
 
+  it('should accept "transparent" value for backgroundColor', () => {
```

Comment:
I'm not sure if this test gives us something meaningful, ideally how do I check that transparent shall remove background and undefined shall apply the base?

Reply:
This just verifies `background-color: 'transparent'` was applied, and can be removed.

Reply:
>I check that transparent shall remove background and undefined shall apply the base?

We can see it from styles in the snap:

1. For `backgroundColor={{ base: 'brand.primary.300', l: undefined }}`:

```
.c0 {
        background-color: hsla(218,89%,51%,0.09);
      }
```

2. For `backgroundColor={{ base: 'brand.primary.300', l: 'transparent' }}`:

```
.c0 {
        background-color: hsla(218,89%,51%,0.09);
      }
      @media screen and (min-width:1024px) {
        .c0 {
          background-color: transparent;
        }
      }
```